Client.xcodeproj/project.pbxproj

```diff
@@ -944,6 +946,20 @@
 			remoteGlobalIDString = 27F443942135E11200296C58;
 			remoteInfo = BraveShareTo;
 		};
+		27FA2D67234E3222004D5D2D /* PBXContainerItemProxy */ = {
```

Comment:
are these project changes still needed?

Reply:
Yeah they showed up when I needed to link Shared and BraveShared to it